app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -81,6 +80,8 @@ def verify_credentials(auth_type = nil, options = {})
          RestClient::ResourceNotFound,
          RestClient::InternalServerError => err
     raise MiqException::MiqUnreachableError, err.message, err.backtrace
+  rescue Hawkular::BaseClient::HawkularException => err
+    raise MiqException::MiqInvalidCredentialsError, "HTTP status #{err.status_code}", err.backtrace
```

Comment:
@moolitayer :+1:  Thanks, fixed. 
Actually this was fixed by the new version of hawkular-client :-) , now it has better `err.message` , so we do not need to override it .